charts/kubelb-ccm/values.yaml

```diff
@@ -16,6 +16,8 @@ kubelb:
   # -- Enable the leader election.
   enableLeaderElection: true
   nodeAddressType: InternalIP
+  # -- Enable to automatically convert Secrets labelled with `kubelb.k8c.io/managed-by: kubelb` to Sync Secrets. This is used to sync secrets from tenants to the LB cluster in a controlled and secure way.
+  enableSecretSynchronizer: false
```

Comment:
Note for the reviewer:

Disabled by default since it required CRUD access for secrets.